docs/UserGuide.md

```diff
@@ -63,7 +55,7 @@ ProductiveNUS is a **desktop application for managing and scheduling your academ
 
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Notes about the command format:**<br>
+**:notebook_with_decorative_cover: Notes about the command format:**<br>
```

Comment:
Is this supposed to be :clipboard: - pointers to note instead? Or if this is intended then we should add this to the icon table as well :)

Reply:
it's supposed to be 📋 Notes about the command format, I forgot to change the icon back to the clipboard. Shall go amend it later :)

Reply:
amended!